metaworld/envs/mujoco/sawyer_xyz/sawyer_xyz_env.py

```diff
@@ -212,6 +236,56 @@ def _target_site_config(self):
         """
         return [('goal', self._target_pos)]
 
+    @property
```

Comment:
I thought these changes were already merged down to the new-reward-functions branch? Is there something weird with rebasing going on here?